package.json

```diff
@@ -73,7 +73,7 @@
     "jquery": "^3.6.0",
     "material-design-icons-iconfont": "^6.1.0",
     "ng2-ui-auth": "^10.0.1",
-    "ngx-markdown": "^13.1.0",
+    "ngx-markdown": "^14.0.1",
```

Comment:
This is a comment for the block starting at line 123, since I can't comment there -- please remove the `ngx-markdown` section.

I needed to add it because we were using ngx-markdown v13; now that you've upgraded to version14, we should no longer need it. I'm hoping that doesn't introduce any audit issues (it shouldn't).